File: Pre-Ileya/Day 2/JavaScript/Level 4/Task6.js
function findLargest(x, y, z){
    let largestNumber = x;
    if (largestNumber < y) {
        largestNumber = y;
    } 
    if (largestNumber < z){
        largestNumber = z;
    }
    return largestNumber;
}

console.log(findLargest(2, 3, 4))
console.log(findLargest(1, 2, 2))
console.log(findLargest(1, 1, 1))

File: Pre-Ileya/Day 2/Java/Level 4/Task6.java
public class Task6{
    public static void main(String[] args){
        findLargest(1, 2, 2));
        findLargest(4, 25, 30));
    }

    public static void findLargest(double x, double y, double z){
        double largestNumber = x;
        if (largestNumber < y) {
            largestNumber = y;
        } 

        if (largestNumber < z){
            largestNumber = z;
        }

        System.out.printf("%.2f", largestNumber);
    }
}
